### PR TITLE
Add per flow periodic ping activation switch

### PR DESCRIPTION
**Pull request by surabujin**
To be able to control how many flows are targeted by periodic pings and to be able to exclude not important flows from periodic pings add `periodic-pings` boolean flag into flow definition.

Only if this flag is set, flow will be periodically pinged.

### CODE REVIEW

File: services/src/atdd/src/test/java/org/openkilda/atdd/FlowCrudBasicRunTest.java [hunk 1]
```diff
@@ -80,13 +81,17 @@ public void successfulFlowCreation(final String flowId, final String sourceSwitc
...
                 new FlowEndpointPayload(destinationSwitch, destinationPort, destinationVlan),
-                bandwidth, false, flowId, null, FlowState.UP.getState());
+                bandwidth, false, false, flowId, null, FlowState.UP.getState());
 
         FlowPayload response = FlowUtils.putFlow(flowPayload);
         assertNotNull(response);
-        response.setLastUpdated(null);
 
-        assertEquals(flowPayload, response);
+        System.out.println(String.format(
```
Comment (abochkarev) [inline services/src/atdd/src/test/java/org/openkilda/atdd/FlowCrudBasicRunTest.java]: It looks like we need to think about logging...

File: services/src/messaging/src/test/java/org/openkilda/messaging/info/flow/FlowInfoDataTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                forwardFlowThread.getFlowId(),
+                new ImmutablePair<>(forwardFlowThread, reverseFlowThread),
+                FlowOperation.CREATE, "unit-test-correlation-id");
+
+        InfoMessage wrapper = new InfoMessage(origin, System.currentTimeMillis(), origin.getCorrelationId());
+        serialize(wrapper);
+        FlowInfoData decoded = (FlowInfoData) ((InfoMessage) deserialize()).getData();
+
+        Assert.assertEquals(
+                String.format("%s object have been mangled in serialisation/deserialization loop",
```
Comment (abochkarev) [inline services/src/messaging/src/test/java/org/openkilda/messaging/info/flow/FlowInfoDataTest.java]: ...object **has** been ...

File: services/src/atdd/src/test/java/org/openkilda/atdd/FlowCrudBasicRunTest.java (hunk 1 above)
Comment (surabujin) [reply services/src/atdd/src/test/java/org/openkilda/atdd/FlowCrudBasicRunTest.java]: We definitely should.

But there is a plan to move out from atdd a lot of test that actually not acceptance tests. So there is a sense to "clean" such issues after planned reorganization.

File: services/src/messaging/src/test/java/org/openkilda/messaging/info/flow/FlowInfoDataTest.java (hunk 2 above)
Comment (surabujin) [reply services/src/messaging/src/test/java/org/openkilda/messaging/info/flow/FlowInfoDataTest.java]: changing

**Pull request merged**